### DISCUSSION TITLE
v3.6.4-R3.4

### DISCUSSION THREAD

**Discussion by RyanMetcalfeInt8** [intel/openvino-plugins-ai-audacity#298] [Announcements]
# Compatibility :wrench:
:warning: This plugin release is *only* compatible with [Audacity 3.6.4](https://github.com/audacity/audacity/releases/tag/Audacity-3.6.4) 64-bit Release for Windows -- so make sure you have this version of Audacity installed! 

## Changes since previous release:
* Audacity compatibility updated from 3.6.3 to 3.6.4

## :warning: If you've previously installed any of our 3.6.X or 3.5.X plugin releases, the recommendation for upgrade is to *not* uninstall the previous version. Instead, just download this version and select *Install no models* from the drop down list. This way, you won't need to re-download all of the models again.

:warning: **Make sure to select an Audacity 3.6.4 location during installation. If you have multiple Audacity versions installed, the installer may select the wrong location by default -- so be mindful of this, and manually adjust this using the *browse* button if needed**. 

:penguin: Although we are only providing an installer for Windows right now, it can be compiled for Linux using [this build procedure](https://github.com/intel/openvino-plugins-ai-audacity/blob/b6290c294040a23081d0ece90ede86c6f0066613/doc/build_doc/linux/README.md).

# Supported Hardware
As this release is built on top of OpenVINO™ 2024.3.0, it will generally support hardware defined in this list: https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html 

# Need help? :raising_hand:
If you encounter any issues during install procedure, reach out to us for help by filing an issue [here](https://github.com/intel/openvino-plugins-ai-audacity/issues). We are happy to help!

<hr /><em>This discussion was created from the release <a href='https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.6.4-R3.4'>v3.6.4-R3.4</a>.</em>

**Comment by hafizsikder** [comment]
Did not work!

**Reply by RyanMetcalfeInt8** [reply]
Hi! I definitely need more details than that if you want some help. Thanks!

**Reply by hafizsikder** [reply]
i followed every step. yet audacity has not detected the plugins. kindly let me know, what do you need to help me!

**Reply by RyanMetcalfeInt8** [reply]
Did you enable 'mod-openvino' inside of Audacity, within Edit->Preferences->Modules? After you do this, you need to close & re-open Audacity.

**Reply by hafizsikder** [reply]
I have not! Let me inform you right away!

**Reply by hafizsikder** [reply]
Thank you so much! You are a lifesaver!

**Reply by RyanMetcalfeInt8** [reply]
You're welcome, enjoy!

**Reply by hafizsikder** [reply]
I faced a new issue! While transcribing in English it does a wonderful job! But it doesn't work in the Bengali language, the font seems in Hindi or something like that. I wonder if you could help me!

**Reply by RyanMetcalfeInt8** [reply]
Might be a couple days, but I can take a look if you're willing to [file a new issue](https://github.com/intel/openvino-plugins-ai-audacity/issues/new/choose) that contains:
1. Link to a short audio snippet (wav)
2. screenshot of transcription setting dialog (that show which model you're using, etc.)
3. screenshot of label track that you get (which has wrong font)
4. A better description of what you *wanted* it to produce.

**Comment by xloat-dreamcondition** [comment]
Is this not available for Mac users?

**Reply by RyanMetcalfeInt8** [reply]
Hello, correct, it is currently only supported on Windows and Linux.

You can take a look at this discussion for Mac support: https://github.com/intel/openvino-plugins-ai-audacity/discussions/274
